OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -15,21 +15,28 @@
 
 #include "absl/strings/str_format.h"
 #include "absl/synchronization/mutex.h"
+#include "absl/time/time.h"
 
 #ifdef __clang__
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
 #endif
 
-#define LOG(format, ...)                                                                \
-  do {                                                                                  \
-    std::string file__ = std::filesystem::path(__FILE__).filename().string();           \
-    std::string file_and_line__ = absl::StrFormat("%s:%d", file__.c_str(), __LINE__);   \
-    if (file_and_line__.size() > 28)                                                    \
-      file_and_line__ = "..." + file_and_line__.substr(file_and_line__.size() - 25);    \
-    std::string formatted_log__ =                                                       \
-        absl::StrFormat("[%28s] " format "\n", file_and_line__.c_str(), ##__VA_ARGS__); \
-    PLATFORM_LOG(formatted_log__.c_str());                                              \
+#define LOG(format, ...)                                                                     \
+  do {                                                                                       \
+    std::filesystem::path path__ = std::filesystem::path(__FILE__);                          \
+    std::string dir__;                                                                       \
+    auto path_it__ = path__.end();                                                           \
+    if (path_it__ != path__.begin() && --path_it__ != path__.begin() &&                      \
+        --path_it__ != path__.begin()) {                                                     \
+      dir__ = path_it__->string();                                                           \
+    }                                                                                        \
+    std::string time__ = absl::FormatTime(absl::Now(), absl::UTCTimeZone());                 \
```

Comment:
How does an example formatted time look like?
I'm afraid it would look like the example given in the docs, `2018-08-06T23:35:32.637472794+00:00`, in which case some stuff could definitely be removed, like the time zone and honestly at least the last three digits of the decimal part.

Reply:
removed timezone and reduced precision to microseconds (6 digits)